crates/cargo-contract/src/cmd/info.rs

```diff
@@ -0,0 +1,121 @@
+//
+// cargo-contract is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with cargo-contract.  If not, see <http://www.gnu.org/licenses/>.
+
+use super::{
+    runtime_api::api::{self,},
+    Client,
+    DefaultConfig,
+};
+use crate::{
+    cmd::{
+        runtime_api::api::runtime_types::pallet_contracts::storage::ContractInfo,
+        ErrorVariant,
+    },
+    name_value_println,
+    DEFAULT_KEY_COL_WIDTH,
+};
+use anyhow::{
+    anyhow,
+    Result,
+};
+use std::fmt::Debug;
+use subxt::{
+    Config,
+    OnlineClient,
+};
+
+#[derive(Debug, clap::Args)]
+#[clap(name = "info", about = "Get infos from a contract")]
+pub struct InfoCommand {
+    /// Address of the contract to display info.
+    #[clap(name = "contract", long, env = "CONTRACT")]
+    contract: <DefaultConfig as Config>::AccountId,
+    #[clap(
+        name = "url",
+        long,
+        value_parser,
+        default_value = "ws://localhost:9944"
+    )]
+    url: url::Url,
+}
+
+impl InfoCommand {
+    pub fn run(&self) -> Result<(), ErrorVariant> {
+        tracing::debug!(
+            "Getting contract information for AccountId {:?}",
+            self.contract
+        );
+
+        async_std::task::block_on(async {
+            let url = self.url.clone();
+            let client = OnlineClient::<DefaultConfig>::from_url(url).await?;
+
+            let info_result = self.fetch_contract_info(&client).await?;
+
+            match info_result {
+                Some(info_result) => {
+                    InfoCommand::print_and_format_contract_info(info_result);
+                }
+                None => {
+                    return Err(anyhow!(
+                        "No contract information was found for account id {}",
+                        self.contract
+                    )
+                    .into())
+                }
+            }
+            Result::<(), ErrorVariant>::Ok(())
+        })
+    }
+
+    async fn fetch_contract_info(&self, client: &Client) -> Result<Option<ContractInfo>> {
+        let info_contract_call =
+            api::storage().contracts().contract_info_of(&self.contract);
+
+        let contract_info_of = client
+            .storage()
+            .at(None)
+            .await?
+            .fetch(&info_contract_call)
+            .await?;
+
+        Ok(contract_info_of)
+    }
+
+    fn print_and_format_contract_info(info: ContractInfo) {
+        let convert_trie_id = hex::encode(info.trie_id.0);
+        name_value_println!(
+            "TrieId:",
+            format!("{}", convert_trie_id),
+            DEFAULT_KEY_COL_WIDTH
+        );
+        name_value_println!(
+            "Code hash:",
+            format!("{}", info.code_hash),
```

Comment:
<img width="1070" alt="image" src="https://user-images.githubusercontent.com/241530/223484502-d6f900f3-f5a4-434b-870a-48e22b39d18c.png">

This is what I'm getting, it should be the unshortened hash here.